zebra/rt_netlink.c

```diff
@@ -2274,6 +2274,28 @@ static int netlink_macfdb_update(struct interface *ifp, vlanid_t vid,
 			    0);
 }
 
+/*
+ * In the event the kernel deletes ipv4 link-local neighbor entries created for
+ * 5549 support, re-install them.
+ */
+static void netlink_handle_5549(struct ndmsg *ndm, struct zebra_if *zif,
+				struct interface *ifp, struct ipaddr *ip)
+{
+	char b[16] = "169.254.0.1";
+
+	if (ndm->ndm_family != AF_INET)
+		return;
+
+	if (!zif->v6_2_v4_ll_neigh_entry)
+		return;
+
+	inet_pton(AF_INET, b, &ipv4_ll);
```

Comment:
nit: this shouldn't be necessary since the global `ipv4_ll` is initialized by `rt_netlink_init()`. The `b` buffer can be removed as well.